main.go

```diff
@@ -0,0 +1,34 @@
+
+package main
+
+import (
+    "./sreality"
+    _ "./models"
+    "./db"
+    "./config"
+    "./email"
+)
+
+
+
+func main() {
+    config := config.New()
+
+    var urlParameters *sreality.UrlParams = sreality.New(config)
+    adverts := urlParameters.MakeRequest()
+
+    db.Storage = db.NewStorage(config)
+    defer db.Storage.Close()
+
+    for _, advert := range *(adverts.GetAdverts()) {
```

Comment:
Main should just read console/terminal arguments, read config, connect to db and call some service/storage method to DO business work or start a server/client.. etc. Move this business logic from main somewhere.